x/harvest/keeper/borrow.go

```diff
@@ -61,6 +71,22 @@ func (k Keeper) ValidateBorrow(ctx sdk.Context, borrower sdk.AccAddress, amount
 			return sdkerrors.Wrapf(types.ErrPriceNotFound, "no price found for market %s", moneyMarket.SpotMarketID)
 		}
 		coinUSDValue := sdk.NewDecFromInt(coin.Amount).Quo(sdk.NewDecFromInt(moneyMarket.ConversionFactor)).Mul(assetPriceInfo.Price)
+
+		// Validate the requested borrow value for the asset against the money market's global borrow limit
```

Comment:
Refactor this validation logic based on `BorrowLimit` params comment

Reply:
https://github.com/Kava-Labs/kava/pull/715/commits/777fc3e26d0db82666294056b9b923a1f0061f92